libchannelserver/ChannelRPCServer.cpp

```diff
@@ -444,6 +496,37 @@ void dev::ChannelRPCServer::onClientHeartbeat(
     }
 }
 
+void dev::ChannelRPCServer::asyncPushChannelMessageHandler(
```

Comment:
this can be a lambda, to that you don't depend on channlserver's point.
you can read below
https://github.com/FISCO-BCOS/FISCO-BCOS/blob/a589e4d7f067a1f26cbe49925f889b2b95844727/libinitializer/RPCInitializer.cpp#L124-L140